README.md

```diff
@@ -22,6 +22,9 @@ If you have any questions or need more information, feel free to reach out to su
 * 3.10
 * 3.11
 
+## Requirements
+Duo_client_python supports python 3.7 and higher
```

Comment:
_technically_ Python should be capitalized

Reply:
This is a tech company's repo. So we will do the technical thing. :P